app/models/Export.php

```diff
@@ -158,7 +159,17 @@ public static function full_yes_data_query()
     {
         $results = DB::select('SELECT u.first_name, u.last_name, u.email,
                           p.birthdate, p.phone, p.address_street, p.address_premise, p.city, p.state, p.zip, p.gender, p.school, p.grade,
-                          a.accomplishments, a.activities, a.participation, a.essay1, a.essay2, a.extra_question_1, a.extra_question_2, a.extra_question_3, a.extra_question_4, a.extra_question_5, a.hear_about, a.test_type, a.test_score, a.gpa,
+                          REPLACE(REPLACE(REPLACE(REPLACE(a.accomplishments,CHAR(10)," "),CHAR(13)," "), CHAR(9)," "), CHAR(44),";") as accomplishments,
```

Comment:
What is happening here? 😳

Reply:
greatttttt question. This escapes a bunch of things like tabs and newlines and commas (commas are replaced with semicolons, everything else with a space) because those things will really muck up the formatting when the CSV is opened in excel (putting one application onto several different lines).

Reply:
Ahhh, gotcha. I don't think it's important for merge, but we should probably do that with a PHP iterator rather than in the database query. Maybe just add a `// TODO:` comment so we remember this next time we're in here?

Reply:
@DFurnes oooo that is something that I do not know about! what are the benefits of doing it that way instead? I'm adding the `todo` now!

Reply:
Just using the most efficient (and clear!) tool for the job. Ideally the SQL query should just be responsible for fetching things from the database, and we can then more easily re-format things once it's in memory with `str_replace` or `preg_replace`.